v1/nix/modules/drv-parts/fetch-pip/interface.nix

```diff
@@ -0,0 +1,85 @@
+{
+  config,
+  lib,
+  ...
+}: let
+  l = lib // builtins;
+  t = l.types;
+in {
+  options.fetch-pip = {
+    hash = l.mkOption {
+      type = t.str;
+      description = ''
+        hash for the fixed output derivation
+      '';
+    };
+    maxDate = l.mkOption {
+      type = t.str;
+      description = ''
+        maximum release date for packages
+        Choose any date from the past.
+      '';
+      example = "2023-01-01";
+    };
+    nameSuffix = l.mkOption {
+      type = t.str;
+      default = "python-requirements";
+      description = ''
+        suffix of the fetcher derivation name
+      '';
+    };
+    onlyBinary = l.mkOption {
+      type = t.bool;
```

Comment:
Currently it is a bool only and it gets automatically translated to pip flags. Should we change that?

Reply:
Don't know; if there's no special reason to handle it like that, maybe we could just leave it to pipFlags?

Reply:
I missed something. The `onlyBinary` does more than just adding flags. It also triggers some checks, because not setting `onlyBinary` is unsafe in cross system scenarios.